setup.py

```diff
@@ -36,9 +46,7 @@ def grep(attrname):
     license='Apache 2.0',
     packages=['arrow'],
     zip_safe=False,
-    install_requires=[
-        'python-dateutil'
-    ],
+    install_requires=install_requires,
     test_suite="tests",
     classifiers=[
```

Comment:
Don't forget to add the new versions the classifiers. Can also add CPython/PyPy:

https://pypi.python.org/pypi?%3Aaction=list_classifiers